data.sql

```diff
@@ -13,3 +13,41 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUE
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,eight_kg) VALUES('Angemon',date '2005-04-12',1,FALSE,-45);
```

Comment:
Please add the project title and project features description in README. This way it looks more professional.

Reply:
added features into readme